packages/core/admin/admin/src/content-manager/components/AttributeFilter/index.js

```diff
@@ -1,38 +1,76 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import PropTypes from 'prop-types';
 import { useIntl } from 'react-intl';
-import useAllowedAttributes from './hooks/useAllowedAttributes';
-import Filters from './Filters';
+import { FilterListURLQuery, FilterPopoverURLQuery, useTracking } from '@strapi/helper-plugin';
+import { Box, Button } from '@strapi/design-system';
+import { Filter } from '@strapi/icons';
+
+import { useAllowedAttributes } from './hooks/useAllowedAttributes';
+
+export function AttributeFilter({ layout, slug }) {
+  const { contentType } = layout;
+  const { metadatas } = contentType;
 
-const AttributeFilter = ({ contentType, slug, metadatas }) => {
   const { formatMessage } = useIntl();
+  const [isVisible, setIsVisible] = useState(false);
+  const buttonRef = useRef();
+  const { trackUsage } = useTracking();
   const allowedAttributes = useAllowedAttributes(contentType, slug);
   const displayedFilters = allowedAttributes.map((name) => {
-    const attribute = contentType.attributes[name];
-    const { type, enum: options } = attribute;
-
-    const trackedEvent = {
-      name: 'didFilterEntries',
-      properties: { useRelation: type === 'relation' },
-    };
-
+    const { type, enum: options } = contentType.attributes[name];
     const { mainField, label } = metadatas[name].list;
 
     return {
       name,
-      metadatas: { label: formatMessage({ id: label, defaultMessage: label }) },
```

Comment:
I don't thin we should do this ... and I don't think it was documented anywhere this was possible.

Reply:
I don't actually understand what this is, i've not worked on it before, can you explain please? 😅

Reply:
oh yeah no that's dumb af because this goes into the DB and we don't use `formatMessage` on anything that goes into the DB.